c2corg_ui/static/js/documentediting.js

```diff
@@ -536,31 +577,52 @@ app.DocumentEditingController.prototype.updateMaxSteps = function(waypointType)
  * @param {Object} outing
  * @private
  */
-
 app.DocumentEditingController.prototype.formatOuting_ = function(outing) {
-  // creating a new outing -> init locales
+  if (this.submit_) {
+    if (outing['locales'][0]['conditions_levels'][0]['level_place'].length > 0) {
```

Comment:
You are super right, but as you stated I would prefer to correct it in the future, when the API will accept conditions_levels as objects or null (this will greatly simplify the code already by itself) and when we decide to tidy up the code and create document-specific controllers, in order to make the documentediting directive more generic.